src/go/rpk/pkg/config/config.go

```diff
@@ -16,6 +16,7 @@ import (
 	"math/big"
 	"os"
 	"path/filepath"
+	fp "path/filepath"
```

Comment:
importing `path/filepath` twice. Is this actually needed?

Reply:
It's not. Good catch. I added the alias because some lines were getting long, but then I forgot about it I guess.